instance/models/mixins/openedx_database.py

```diff
@@ -48,11 +49,52 @@ def mysql_database_name(self):
         return self.database_name
 
     @property
-    def mysql_database_names(self):
-        """
-        List of mysql database names
-        """
-        return [self.mysql_database_name]
+    def mysql_databases(self):
+        """
+        List of mysql databases
+        """
+        return [
+            {
+                "name": self._get_mysql_database_name("ecommerce"),
+                "user": self._get_mysql_user_name("ecomm001"),
```

Comment:
I'm not a fan of these usernames, even though I think the inspiration comes from the defaults in edx/configuration. I don't see the point of the `001` suffix. I would prefer to remove it unless that would cause issues.